tests/api/test_processes.py

```diff
@@ -289,3 +374,35 @@ async def test_post_process_id_not_found_in_config(
         data=json.dumps(process_payload_valid_wf_name_not_in_config),
     )
     assert response.status_code == 404
+
+
+@pytest.mark.asyncio
+async def test_post_payload_cache(test_client, process_payload_cache):
+    response_one = test_client.post(
+        "/processes/mirror/execution", data=json.dumps(process_payload_cache)
+    )
+    assert response_one.status_code == 201
+
+    response_two = test_client.post(
+        "/processes/mirror/execution", data=json.dumps(process_payload_cache)
+    )
+    assert response_two.status_code == 200
+
+
+@pytest.mark.asyncio
+async def test_post_payload_cache_key_change(
+    test_client, process_payload_cache_key_change, workflow_config_cache_key_change
+):
+    response_one = test_client.post(
+        "/processes/mirror/execution", data=json.dumps(process_payload_cache_key_change)
+    )
+    assert response_one.status_code == 201
+
+    test_client.app.state.workflows = Workflows.from_yaml(
+        workflow_config_cache_key_change
+    )
+
+    response_two = test_client.post(
+        "/processes/mirror/execution", data=json.dumps(process_payload_cache_key_change)
+    )
+    assert response_two.status_code == 201
```

Comment:
Can you help me understand the idea of this test? Under what scenario can the workflow config change like this? I believe the only way to change the loaded workflow configs is to restart the service entirely, no?

I mean, don't get me wrong, I think having the ability to reload the workflow config file would be _nice_. That said, config updates are probably not going to happen often, so from an operational perspective simply depending on a service restart is probably safer and adequate, at least until we have a use-case dictating otherwise.